experimental/char16trie/src/char16trie.rs

```diff
@@ -75,8 +75,7 @@ fn skip_node_value(pos: usize, lead: u16) -> usize {
 /// For more information:
 /// - [ICU4C UCharsTrie](https://unicode-org.github.io/icu-docs/apidoc/released/icu4c/classicu_1_1UCharsTrie.html)
 /// - [ICU4J CharsTrie](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/util/CharsTrie.html) API.
-#[cfg_attr(feature = "serde", derive(serde::Deserialize))]
-#[cfg_attr(feature = "serde_serialize", derive(serde::Serialize))]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
```

Comment:
Nit: I'm not opposed to combining these features on utils crates, but you should mention this in the PR title